common/dynamicconfig/constants.go

```diff
@@ -2354,4 +2353,9 @@ settings for controlling remote activity concurrency for delete namespace workfl
 		20000,
 		`MaxUserMetadataDetailsSize is the maximum size of user metadata details payloads in bytes.`,
 	)
+	WorkflowDeduplicationGracePeriod = NewGlobalDurationSetting(
+		"system.workflowDeduplicationGracePeriod",
+		1*time.Second,
+		`WorkflowDeduplicationGracePeriod is used for timing how soon users can create new workflow with the same runID.`,
```

Comment:
replace runID with WorkflowID

Reply:
Maybe rename to system.workflowIdReuseMinimalInterval.

Reply:
Also should consider combine this key with history.continueAsNewMinInterval

Reply:
done for renaming. As for different constant - while "min interval" is a common between them, I feel they are used in a quite different manner. Lets not do that now. Also it is per namespace.

Reply:
I think this should be per-namespace as well, it give us more flexible in case we need to special case handle certain namespace. Having too many config keys for similar purpose is not great. Ok to do that in a separate PR through.